src/main/java/ladder/Results.java

```diff
@@ -0,0 +1,32 @@
+package ladder;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.stream.Collectors.toList;
+
+public class Results {
+    private final List<Result> results;
+
+    public Results(List<String> results) {
+        this.results = results.stream()
+                .map(Result::new)
+                .collect(toList());
+    }
+
+    public Results(String... results) {
```

Comment:
```suggestion
    public Results(String... results) {
        this(Arrays.asList(results));
    }
```
최후의 수단으로 위 생성자를 사용해야한다면 다음과 같이 정의가 되어야하지 않을까 싶어요.